base/challenge-skeleton/config/advanced/network/network.yaml

```diff
@@ -15,11 +15,3 @@ apiVersion: "v1"
 kind: "Service"
 metadata:
   name: "chal"
-#spec:
```

Comment:
You don't think the example here is useful?

Reply:
They are a bit confusing, since they are explaining how to do something that is not needed (map port 1 to 1337). I think we should have a README here instead (and in the other folder) explaining what docs to read and an example.